### PR TITLE
Add custom favicon property

### PR DESCRIPTION
**Pull request by shershen08**

### CODE REVIEW

**Comment by joshiste** [comment]
please add documentation for the properties in spring-boot-admin-docs/spring-boot-admin-docs/src/main/asciidoc/server.adoc
And in spring-boot-admin-docs/spring-boot-admin-docs/src/main/asciidoc/customizing.adoc a short example on how to customize the favicon.

File: spring-boot-admin-docs/src/main/asciidoc/customizing.adoc [hunk 1]
```diff
@@ -95,3 +95,11 @@ Either you just add the image to your jar-file in `/META-INF/spring-boot-admin-s
 or you must ensure yourself that the image gets served correctly (e.g. by registering your own `ResourceHandler`)
 
 - **spring.boot.admin.ui.title**: Use this option to customize the browsers window title.
+
+=== Customizing Page Favicon ===
+You can show custom browser favicon by using following configuration properties:
```
[the remark this answers is not in the document]
Comment (shershen08) [reply spring-boot-admin-docs/src/main/asciidoc/customizing.adoc]: I understand your point, but could you give me a pointer - what could be a proper place?

[the remark this answers is not in the document]
Comment (joshiste) [reply spring-boot-admin-docs/src/main/asciidoc/customizing.adoc]: one that works out of the box?

[the remark this answers is not in the document]
Comment (joshiste) [reply spring-boot-admin-docs/src/main/asciidoc/customizing.adoc]: e.g. `/META-INF/spring-boot-admin-server-ui/assets/img/custom-favicon.png`

[the remark this answers is not in the document]
Comment (shershen08) [reply spring-boot-admin-docs/src/main/asciidoc/customizing.adoc]: added

**Comment by joshiste** [comment]
merged with a811f3e996700b98dbc6d81c2b5303c0e31c88c1 
polished with f660f29417a3eb1329226347b171af3d80f224cf

**Pull request closed without merging**